### PR TITLE
Bump version of package

### PR DESCRIPTION
**Pull request by turt2live**
This tries to maintain the version number we're based off of while also changing it for us to use.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-gemini-scrollbar",
-  "version": "2.1.5",
+  "version": "2.1.5-1",
```
Comment (jryans) [inline package.json]: Maybe this is too pedantic, but looking at https://semver.org/, it appears that a `-<stuff>` suffix makes it a pre-release version, and thus lower precedence than the normal version... Is that what we want?

Comment (turt2live) [reply package.json]: @jryans I guess we want `+1` (for a build identifier)?

Comment (turt2live) [reply package.json]: In the last 4 minutes I've convinced myself that +1 is the right answer - will try that

Comment (jryans) [reply package.json]: Seems probably right, let's try it!

**Review by jryans — APPROVED** [review]

**Pull request merged**